src/main/java/com/mehmet_27/punishmanager/Punishment.java

```diff
@@ -92,40 +92,49 @@ public long getEnd() {
     }
 
     public String getDuration() {
+        Configuration messages = PunishManager.getInstance().getConfigManager().getMessages();
         long currentTime = new Timestamp(System.currentTimeMillis()).getTime();
         long diff = (getEnd() - currentTime) / 1000;
+
         //Getting time formats
-        String monthFormat = PunishManager.getInstance().getConfigManager().getMessages().getString("main.timelayout.month");
-        String weekFormat = PunishManager.getInstance().getConfigManager().getMessages().getString("main.timelayout.week");
-        String dayFormat = PunishManager.getInstance().getConfigManager().getMessages().getString("main.timelayout.day");
-        String hourFormat = PunishManager.getInstance().getConfigManager().getMessages().getString("main.timelayout.hour");
-        String minuteFormat = PunishManager.getInstance().getConfigManager().getMessages().getString("main.timelayout.minute");
-        String secondFormat = PunishManager.getInstance().getConfigManager().getMessages().getString("main.timelayout.second");
+        String monthFormat = messages.getString("main.timelayout.month");
+        String weekFormat = messages.getString("main.timelayout.week");
+        String dayFormat = messages.getString("main.timelayout.day");
+        String hourFormat = messages.getString("main.timelayout.hour");
+        String minuteFormat = messages.getString("main.timelayout.minute");
+        String secondFormat = messages.getString("main.timelayout.second");
 
         String months = String.valueOf(diff / 60 / 60 / 24 / 7 / 4);
         String weeks = String.valueOf(diff / 60 / 60 / 24 / 7 % 4);
         String days = String.valueOf(diff / 60 / 60 / 24 % 7);
         String hours = String.valueOf(diff / 60 / 60 % 24);
         String minutes = String.valueOf(diff / 60 % 60);
         String seconds = String.valueOf(diff % 60);
-        // months
-        if (diff > 60 * 60 * 24 * 7 * 4) {
-            return (monthFormat + " " + weekFormat + " " + dayFormat + " " + hourFormat + " " + minuteFormat + " " + secondFormat).replaceAll("%mo%", months).replaceAll("%w%", weeks).replaceAll("%d%", days).replaceAll("%h%", hours).replaceAll("%m%", minutes).replaceAll("%s%", seconds);
+
+        if (diff > MONTH) {
+            return String.format("%s %s %s %s %s %s", monthFormat, weekFormat, dayFormat, hourFormat, minuteFormat, secondFormat).
+                    replaceAll("%mo%", months).
+                    replaceAll("%w%", weeks).
+                    replaceAll("%d%", days).
+                    replaceAll("%h%", hours).
+                    replaceAll("%m%", minutes).
+                    replaceAll("%s%", seconds);
```

Comment:
Just an example, didn't touch another lines.

Reply:
It's good to know that.

Reply:
It's good to know that.